src/feedback/types.ts

```diff
@@ -6,9 +6,14 @@ export type FeedbackPlacement =
   | "top-right"
   | "top-left";
 
+export type Offset = {
+  left?: string | number;
+  top?: string | number;
```

Comment:
`top` and `left` are ambiguous, since this is in a positioning context where we already place the box in one of four corners. I think we either need to go more abstract, like the `x` and `y` from the docs or `horisontal`/ `vertical`

Reply:
Felt the opposite before actually, I've seen `y` mean 'from the top' or 'from the bottom' in different contexts. I've reverted back to x/y though because of the implementation change though